ci/cudf_pandas_scripts/fetch_pandas_versions.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import requests
+
+def get_pandas_versions():
+    url = "https://pypi.org/pypi/pandas/json"
+    response = requests.get(url)
+    data = response.json()
+    versions = data['releases'].keys()
+
+    # Filter out pre-releases and major versions
+    minor_versions = sorted(set(
```

Comment:
IMO we might as well install packaging and use `packaging.version.parse` to manipulate these